VENTouchLock/Controllers/VENTouchLockSplashViewController.m

```diff
@@ -100,9 +100,14 @@ - (void)showTouchID
 
 - (void)showPasscodeAnimated:(BOOL)animated
 {
-    [self presentViewController:[[self enterPasscodeVC] embeddedInNavigationController]
-                                            animated:animated
-                                          completion:nil];
+    VENTouchLockEnterPasscodeViewController *enterPassCodeViewController;
```

Comment:
This should be of type ```UIViewController``` instead of ```VENTouchLockEnterPasscodeViewController``` otherwise setting it to ```[[self enterPasscodeVC] embeddedInNavigationController];``` creates a compiler warning.